src/simcommand/package.scala

```diff
@@ -7,7 +7,7 @@ package object simcommand {
     * This class represents an RTL simulation command and its return value
     * @tparam R Type of the command's return value
     */
-  sealed abstract class Command[+R] {
```

Comment:
Is there a reason we can't keep Command sealed? We would like the compiler to warn us on incomplete matches